data/cadastre/parcels/index.html

```diff
@@ -163,7 +163,7 @@ <h5 id="UtahParcels-downloads">Downloads</h5>
         </tr>
         <tr>
           <td>Kane</td>
-          <td><a href="https://opendata.gis.utah.gov/datasets/utah-kane-county-parcels">January 2022</a></td>
```

Comment:
**issue**: this shouldn't be included. What did you do to pick up these commits?